aten/src/ATen/native/quantized/cpu/qnnpack_utils.h

```diff
@@ -11,12 +11,16 @@ struct QnnpackOperatorDeleter {
 };
 struct PackedLinearWeightsQnnp {
   std::unique_ptr<qnnpack::PackBMatrix> w;
+  at::Tensor orig_weight;
```

Comment:
I'd add a comment explaining why we do this (that implementing proper unpack function would be an alternative. Also, mention that we could easily optimize the memory by not storing the original weights in the mobile case